numalogic/connectors/_config.py

```diff
@@ -52,7 +52,7 @@ class DruidFetcherConf:
     dimensions: list[str] = field(default_factory=list)
     aggregations: dict = field(default_factory=dict)
     group_by: list[str] = field(default_factory=list)
-    pivot: Pivot = field(default_factory=lambda: Pivot())
+    pivot: Optional[Pivot] = field(default_factory=lambda: Pivot())
```

Comment:
Shouldn't we remove the default factory then if this is optional?

Reply:
I don't wanna impact FCi and hence, I did this.
Wdyt?